tests/args/input.txt

```diff
@@ -0,0 +1 @@
+Main.elm hello world
```

Comment:
Because `run-elm` input is a bit more sophisticated now, I switched to a compulsory `input.txt` instead of a former optional `args.json`